src/frontend/src/main.tsx

```diff
@@ -5,11 +5,17 @@ import { RouterProvider, createBrowserRouter } from 'react-router-dom';
 import routes from 'navigation/RouterConfig';
 
 import 'styles/index.scss';
+import { createClient } from '@supabase/supabase-js';
 
 const router = createBrowserRouter(routes);
 
 ReactDOM.createRoot(document.getElementById('root') as HTMLElement).render(
-	<React.StrictMode>
-		<RouterProvider router={router} />
-	</React.StrictMode>
+    <React.StrictMode>
+        <RouterProvider router={router} />
+    </React.StrictMode>
+);
+
+export const supabase = createClient(
```

Comment:
We’ll probably want to create a subasebaseContext to avoid referencing main all the time, but this looks easy enough

Reply:
Cool!

Reply:
I'll move the API Key into the configuration folder.